clients/ckan_client.py

```diff
@@ -113,7 +114,7 @@ class CkanError(Exception):
     app = create_app()
     with app.app_context():
         ckan_cli = init_ckan(apikey=os.getenv('CKAN_APIKEY'))
-        dataset_id = "3963fdf5-8915-448d-b8dd-9beca9c04a35"
-        fetch_workflow_state(ckan_cli, dataset_id)
+        dataset_id = "switzerland-country-estimates-2022"
+        state = fetch_workflow_state(ckan_cli, dataset_id)
```

Comment:
Does this not break linting?  It looks like you create a variable and never use it.

Reply:
no, it's fine.